stores/models.py

```diff
@@ -1,3 +1,32 @@
 from django.db import models
+from django.contrib import get_user_model
+from final_project import CustomModel
 
+AuthUserModel = get_user_model
+
+
+"""These are the product models, image, charfield is a textbase databasecolumn"""
 # Create your models here.
+
+class Product(CustomModel):
+    name = models.CharField(max_length=255)
+    description = models.TextField()
+    size = models.CharField(choices=ITEM_SIZES,max_length=1)
+    color =models.CharField(max_length=255,unique=True)
+    price = models.DecimalField(max_digits=5,decimal_places=2,default=0.00)
+    quantity = models.IntegerField()
+
+
+ITEM_SIZES = (
+            ('S','Small'),
+            ('M','Medium'),
+            ('L','Large'),
+            )
+
+class Category(CustomModel):
+    name = models.CharField(max_length=255)
+
+class Image(CustomModel):
+    product_id = models.ForeignKey(Product,on_delete=CASCADE)
```

Comment:
No, you don't have to specify `product_id`. `product` is enough. `product_id` will be automatically generated by Django.

Reply:
why is product_id generated? i thought only image_id is generated. is that because it's part of product app?

Reply:
I think i know, product doesn't need to include "id" in the name - product is product id. I corrected it now.